### PR TITLE
fix(RHINENG-1454): handle bugs

### PR DESCRIPTION
**Pull request by johnsonm325**
- When viewing the Task Details in the Activity Tab, then I will see the name given to the task in the task details and I see the default task name below it
- In Run Task MOdal, disable the Execute task button when the task name field is empty
- also:

click Run task
enter empty name
cancel modal
click Run task
The Task name stays empty, but this PR fixes that.

### CODE REVIEW

File: src/SmartComponents/ActivityTable/__tests__/ActivityTable.tests.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ describe('ActivityTable', () => {
     jest.clearAllMocks();
   });
 
-  it('should render correctly', () => {
+  it.skip('should render correctly', () => {
```
Comment (bastilian) [inline src/SmartComponents/ActivityTable/__tests__/ActivityTable.tests.js:39]: Why do all these need to be skipped?

Comment (johnsonm325) [reply src/SmartComponents/ActivityTable/__tests__/ActivityTable.tests.js:39]: I didn't mean for them all to be skipped. I forgot to change them. Good catch.

**Comment by johnsonm325** [comment]
@bastilian you can take another look

**Review by bastilian — APPROVED** [review]
LGTM! Thank you @johnsonm325!

**Pull request merged**